src/actions/remoteConfigActions.js

```diff
@@ -41,3 +47,28 @@ export const loadRemoteConfigAction = () => {
       .catch(e => { reportLog('Failed to fetch feature flags or initialize with defaults', e); });
   };
 };
+
+export const setUserPropertiesAction = () => (
+  dispatch: Dispatch,
+  getState: GetState,
+) => {
+  const { accounts: { data: accounts } } = getState();
+
+  const properties = {
+    ens: getEnsName(accounts),
+    network: getEnv().NETWORK_PROVIDER,
+    buildNumber: DeviceInfo.getBuildNumber(),
+    appVersion: DeviceInfo.getVersion(),
+  };
+
+  return firebaseAnalytics
+    .setUserProperties(properties)
+    .catch((error) => {
+      reportLog('Firebase Analytics: Failed to set user properties', { error });
+    });
+};
+
+export const loadRemoteConfigWithUserPropertiesAction = () => async (dispatch: Dispatch) => {
```

Comment:
It seems that there is no logical connection between two actions, as first sets user context info for firebase analytics while second loads firebase remote config. If there are indeed not connected it would be better to keep them separate.